src/_listObject.scss

```diff
@@ -0,0 +1,226 @@
+
+@import "./listDefinitions";
+
+.#{$block} {
+  &--object {
+    .#{$block}__item {
+      @include padding-top-bottom(1rem);
+
+      height: 100%;
+      width: 100%;
+      box-sizing: border-box;
+      border-bottom: $fd-list-object-border-bottom;
+    }
+
+    .#{$block}-optional {
+      width: 100%;
+      padding-bottom: 0.75rem;
+
+      &__text {
+        @include fd-ellipsis();
+        @include fd-reset();
+
+        display: block;
+        width: 50%;
+        color: $fd-list-object-font-color;
+
+        @include fd-rtl() {
+          float: right;
+          width: 100%;
+          padding-bottom: 0.75rem;
+        }
+      }
+    }
+
+    .#{$block}__content {
+      flex-grow: 1;
+    }
+
+    .#{$block}__object {
+      width: 100%;
+
+      &--top-row {
+        display: flex;
+        vertical-align: top;
+      }
+
+      &--top-left {
+        flex: 3 3 10%;
```

Comment:
This will most probably cause problems in IE11

Reply:
updated.